catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/PipelineRepository.java

```diff
@@ -359,7 +359,9 @@ private void updateTaskDescription(Task origTask, Task updatedTask) throws JsonP
         return;
       }
       // Don't record a change if descriptions are the same
-      if (!origTask.getDescription().equals(updatedTask.getDescription())) {
+      if (origTask != null
```

Comment:
@vivekratnavel we should also check if the updated task description is also null. There is no point in recordChange if the updatedTask doesn't have any description